doc/doc_checklists/README.rst

```diff
@@ -0,0 +1,113 @@
+
+Restructured Text
```

Comment:
Why a general intro do RST?

Reply:
@heplesser  Thought this maybe useful information since people are generally more familiar with markdown.

Reply:
Makes sense. But then I would suggest either changing the filename to something like `rst_intro.rst` or to put some information about the contents of doc_checklists first in the file. When I find a README file in a directory, I usually expect information about the purpose and content of the directory.